library/src/main/java/com/getstream/sdk/chat/adapter/DateSeparatorViewHolder.kt

```diff
@@ -0,0 +1,72 @@
+package com.getstream.sdk.chat.adapter
+
+import android.text.format.DateUtils
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.updateLayoutParams
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.adapter.MessageListItem.DateSeparatorItem
+import com.getstream.sdk.chat.enums.Dates
+import com.getstream.sdk.chat.view.MessageListViewStyle
+import io.getstream.chat.android.client.models.Channel
+import java.util.Date
+
+class DateSeparatorViewHolder(
+    resId: Int,
+    viewGroup: ViewGroup,
+    private val style: MessageListViewStyle
+) : BaseMessageListItemViewHolder<DateSeparatorItem>(resId, viewGroup) {
+
+    private val tv_date: TextView = itemView.findViewById(R.id.tv_date)
+    private val iv_line_right: ImageView = itemView.findViewById(R.id.iv_line_right)
+    private val iv_line_left: ImageView = itemView.findViewById(R.id.iv_line_left)
+
+    private var messageListItem: DateSeparatorItem? = null
```

Comment:
It seems that there is not any point to store such property in class

Reply:
Nice catch, I totally missed that it was unused.